changelog/fix-eva-automated-tests-11-2024

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: tweak
+Comment: Fixed tests only
+
```

Comment:
```suggestion

```

We don't need change log for tests.

Reply:
this changelog is from [this PR](https://github.com/the-events-calendar/tribe-common/pull/2290/files)